tests/src/FunctionalJavascript/EntityVersionSettingsTest.php

```diff
@@ -229,7 +233,7 @@ protected function assertFieldSelectOptions(string $name, array $expected_option
     });
     sort($options);
     sort($expected_options);
-    $this->assertIdentical($options, $expected_options);
+    $this->assertSame($options, $expected_options);
```

Comment:
As we've changed this line, let's swap the parameters. In PHPUnit the expectation comes first.